app/frontend/package.json

```
@@ -40,6 +40,17 @@
       "prettier/@typescript-eslint",
       "prettier/react"
     ],
+    "plugins": [
+      "eslint-plugin-simple-import-sort"
```

```suggestion
      "simple-import-sort"
```
Apparently you can omit the `eslint-plugin` bit according to their README

---

I'm going to skip it for now so you don't have to re-approve, feel free to change it upstream with another PR.